src/app/_components/BreadCrumbsWrapper.tsx

```diff
@@ -8,10 +8,16 @@ import { PATHS } from '@/constants/paths'
 
 export function BreadCrumbsWrapper() {
   const pathname = usePathname()
+  const allPaths = Object.values(PATHS).map((route) => route.path) as string[]
 
   if (pathname === PATHS.HOME.path) {
     return null
   }
 
+  // this is to exclude breadcrumbs in not-found.tsx
```

Comment:
I started a discussion on Slack on whether we should only include `Breadcrumbs` on individual blog posts, ecosystem projects and events pages. That is, pages that aren't standalone. This implementation will change pending on that outcome.

Reply:
@mirhamasala based on Friday's discussion - just to confirm, we agreed on not having `Breadcrumbs` on 404?